client/death/FormPage.js

```diff
@@ -146,7 +116,7 @@ export default class IndexPage extends React.Component {
             <div className="txt">
               <label htmlFor="text" className="txt-l">Owner First Name</label>
```

Comment:
`htmlFor` attributes need to match the `id`s of the `input` elements, here and below.